core-plugins/docs/File-batchsink.md

```diff
@@ -0,0 +1,42 @@
+# File Sink
+
+
+Description
+-----------
+Writes to a filesystem in text, avro, or parquet format.
+
+For the text format, each record is written out as delimited text.
+Non-string fields will be converted to strings using their ``toString()`` Java method,
+so fields should be limited to the string, long, int, double, float, and boolean types.
```

Comment:
should we also document the behavior for parquet or avro format here.

Reply:
done